azure-devops/projects/io-backend-projects/io-functions-sign.tf

```
@@ -0,0 +1,80 @@
+variable "io-functions-sign" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "io-functions-sign"
+      branch_name    = "main"
+      pipelines_path = ".azdo"
```

```suggestion
      pipelines_path = ".devops"
```
not blocking but better use a standard naming convention

---

I thought ".azdo" was the standard convention since it is used in microsoft repositories (like https://github.com/microsoft/terraform-provider-azuredevops/tree/main/.azdo)